mne/viz/evoked.py

```diff
@@ -217,6 +217,21 @@ def _plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
         If True, draw at the end.
     """
     import matplotlib.pyplot as plt
+    _check_option('spatial_colors', spatial_colors, [True, False, 'auto'])
+
+    # If spatial colors is set to 'auto' check if picks is
+    # greater than 1. If picks is less than or equal to 1
+    # then spatial_colors is set to 'False'.
+    # If picks is greater than 1, set spatial colors
+    # True if channel locations exists False otherwise.
+    if spatial_colors == 'auto':
+        if picks and len(picks) == 1:
+
+            # If picks is not None and just one single channel
+            # spatial_colors="auto" becomes False
+            spatial_colors = False
+        else:
+            spatial_colors = _check_ch_locs(info=evoked.info)
```

Comment:
The comment doesn't match the code, no? I don't see a check for `len(picks) > 1` or `<= 1`.

Reply:
I would use a private helper function here something like

`spatial_colors = _check_spatial_colors(info, spatial_colors)`

that would return True or False and that can be reused in any place necessary